### PR TITLE
dispatch :filter-activities-by-search-term from :get-activity-metadat…

### PR DESCRIPTION
**Pull request by eemshi**
…a-successful when :search route-param exists

### CODE REVIEW

File: src/cljs/owlet_ui/events.cljs [hunk 1]
```diff
@@ -333,7 +333,7 @@
                                           branches-template)
                                     (into {}))]
       (when-let [route-params (get-in db [:app :route-params])]
-        (let [{:keys [activity branch]} route-params]
+        (let [{:keys [activity branch search]} route-params]
```
Comment (dviramontes) [inline src/cljs/owlet_ui/events.cljs:336]: 💯

File: src/cljs/owlet_ui/events.cljs [hunk 2]
```diff
@@ -343,7 +343,9 @@
                         :skills skills
                         :activities-by-branch activities-by-branch
                         :activity-titles activity-titles
-                        :activity-platforms platforms-nomalized)))))
+                        :activity-platforms platforms-nomalized)))
+          (when search
```
Comment (dviramontes) [inline src/cljs/owlet_ui/events.cljs:347]: 👏

**Review by dviramontes — APPROVED** [review]
💯

**Review by ZadenRB — APPROVED** [review]
LGTM 👍

**Pull request merged**